fratikcoiny/src/main/java/pl/fratik/fratikcoiny/libs/chinczyk/Chinczyk.java

```diff
@@ -1751,8 +1759,9 @@ public void setControlHook(InteractionHook controlHook) {
                 }
                 try {
                     if (Objects.equals(this.controlHook, controlHook)) this.controlHook = null;
-                    Message msg = new MessageBuilder(t.get(getLanguage(), "chinczyk.control.expired")).build();
-                    controlHook.editOriginal(msg).complete();
+                    MessageEditBuilder messageEditBuilder = new MessageEditBuilder();
```

Comment:
mogą zostać przyciski
```suggestion
                    MessageEditBuilder messageEditBuilder = new MessageEditBuilder().setReplace(true);
```